src/GovUk.Education.ExploreEducationStatistics.Content.Model/Database/ContentDbContext.cs

```diff
@@ -535,6 +535,15 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
             modelBuilder.Entity<KeyStatisticDataBlock>()
                 .ToTable("KeyStatisticsDataBlock");
+
+            modelBuilder.Entity<KeyStatisticDataBlock>()
+                .HasOne<DataBlock>(ks => ks.DataBlock)
+                .WithMany()
+                // WARN: This is necessary - otherwise a cascade delete occurs when an associated data block is removed.
+                // This cascade delete _only_ removes the KeyStatisticsDataBlock entry, leaving a KeyStatistics table
+                // entry.
```

Comment:
I don't understand this comment, because there's no cascade delete now.
Should it be `// That cascade delete _only_ removed`?

Reply:
I've updated the comment - hopefully clearer now.